tests/conftest.py

```diff
@@ -11,7 +11,7 @@
 import pytest
 import xarray as xr
 from xclim.testing.helpers import test_timeseries as timeseries
-from xclim.testing.utils import open_dataset as _open_dataset
+from xclim.testing.utils import open_dataset
```

Comment:
```suggestion
from xclim.testing.utils import nimbus as _nimbus
```
I'm trying to deprecate this function entirely. In the latest `xclim`, this is a wrapper around `nimbus`. Will add a better way forward below.